src/jitlayers.cpp

```diff
@@ -2245,10 +2245,15 @@ void jl_merge_module(orc::ThreadSafeModule &destTSM, orc::ThreadSafeModule srcTS
 
 std::unique_ptr<TargetMachine> JuliaOJIT::cloneTargetMachine(std::string targettriple, std::string cpu) const
 {
+    llvm::InitializeAllTargets();
+    llvm::InitializeAllTargetMCs();
```

Comment:
I had them. I just removed them to understand check fails.

Reply:
The LLVM we ship can't compile all supported architectures, only the native one and some gpu ones,